packages/react-devtools-core/src/standalone.js

```diff
@@ -44,16 +46,18 @@ function reload() {
   ReactDOM.unmountComponentAtNode(node);
   node.innerHTML = '';
   setTimeout(() => {
-    ReactDOM.render(<Panel showHiddenThemes={true} {...config} />, node);
+    panel = ReactDOM.render(<Panel showHiddenThemes={true} {...config} />, node);
```

Comment:
FYI `showHiddenThemes` was moved into the `config` in another PR. (Sorry! I Should have put it there to begin with.)

Should be an easy fix to just drop in:
```jsx
ReactDOM.render(<Panel {...config} />, node);
```